connexion/decorators/validation.py

```diff
@@ -114,6 +116,9 @@ def __call__(self, function):
 
         @functools.wraps(function)
         def wrapper(*args, **kwargs):
+            if 'application/json' not in self.consumes:
```

Comment:
Again the wrong check. And I would put the validation logic inside the if to avoid repeating `function(*args, **kwargs)`